### ISSUE TITLE
Timezone issue when using Container

### ISSUE THREAD

**Issue by MikeTheTux** [LycheeOrg/Lychee-Docker#115]
In https://github.com/LycheeOrg/Lychee/issues/1206 we identified an issue with the time zone configuration, when using the latest container from here.

At the end of all the debugging we figured out that it is not sufficient to set the correct timezone via the `PHP_TZ` docker environment variable only. It needs to be written also as `TIMEZONE` into `.env`.

I guess [inject.sh](https://github.com/LycheeOrg/Lychee-Docker/blob/master/inject.sh) should push the info into `/etc/php/8.1/cli/php.ini` as well as `/conf/.env`. Maybe `/etc/php/8.1/fpm/php.ini` as well.

Details can be read in https://github.com/LycheeOrg/Lychee/issues/1206.

**Comment by d7415** [comment]
Thanks! Because of the way this is handled, I've left `PHP_TZ` and `TIMEZONE` separate, so you'll need to specify `-e TIMEZONE=Europe/Berlin` (or, as you have, manually edit the .env). This seems to override the PHP one anyway.

I have added the fpm php.ini as you suggested - That seems a foolish oversight!

Fixed in 108a1ef8ae0ae388edcfc5fd6f8d08a85b06d3b9. `:dev` should be updated this evening. I won't update the `v4.4.0` image for now, but can if there's demand for it.
